python/lsst/ts/watcher/rules/under_pressure.py

```diff
@@ -0,0 +1,163 @@
+# This file is part of ts_watcher.
+#
+# Developed for Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__all__ = ["UnderPressure"]
+
+import yaml
+
+from .base_ess_rule import BaseEssRule
+
+
+class UnderPressure(BaseEssRule):
+    """Check for low pressure.
+
+    This rule only reads ESS telemetry topics.
+
+    Parameters
+    ----------
+    config : `types.SimpleNamespace`
+        Rule configuration, as validated by the schema.
+
+    Notes
+    -----
+    The alarm name is f"UnderPressure.{name}".
+
+    Like most rules based on data from the ESS CSC: this uses
+    `FilteredTopicField` and its ilk, because a given topic may be output
+    for more than one sensor (e.g. there may be two pressure sensors
+    or two 4-channel pressure sensors connected to the same CSC)
+    where the data is differentiated by the value of the sensorName field.
+    """
+
+    def __init__(self, config):
+        super().__init__(
+            config=config,
+            rule_name=f"UnderPressure.{config.name}",
+            topic_attr_name="tel_pressure",
+            field_name="pressure",
+            is_indexed=True,
+            big_is_bad=False,
+            units="Pa",
+            value_format="0.2f",
+            sensors=config.pressure_sensors,
+        )
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+$schema: http://json-schema.org/draft-07/schema#
+description: Configuration for UnderPressure rule.
+type: object
+properties:
+  name:
+    description: System being monitored, e.g. "MTM1M3 Compressor".
```

Comment:
Since this rule only applies to ESS, it would make sense to use an ESS example here but of course it is fine as it is.

Reply:
I'll tweak it.